config/redirects

```diff
@@ -160,13 +161,18 @@ raw: docs/ecosystem/tools/http-interfaces -> ${devhub}/
 raw: docs/ecosystem/security/client-side-field-level-encryption-guide -> https://www.mongodb.com/docs/manual/core/csfle/
 raw: docs/ecosystem/drivers -> ${base}/
 raw: docs/ecosystem/drivers/scala -> ${base}/scala/
-raw: docs/ecosystem/drivers/ruby -> ${base}/ruby/
+raw: docs/ecosystem/drivers/ruby -> https://www.mongodb.com/docs/ruby-driver/current/
 raw: docs/ecosystem/drivers/python -> ${base}/python/
 raw: docs/ecosystem/drivers/pymongo/ -> ${base}/pymongo/
-raw: docs/ecosystem/drivers/perl/ -> ${base}/
+raw: docs/ecosystem/drivers/perl/ -> ${base}/community-supported-drivers/
```

Comment:
Show the community supported drivers page which includes the sunset perl driver.